Documentation/Manual/Texinfo/extensions.texi

```diff
@@ -227,78 +227,86 @@ T
-represented as a rectangular pattern and follows pattern
-protocol. Pixel values are represented as 32-bit RGBA numbers.
+display images on the screen. An image is a rectangular object and is
+represented as a rectangular pattern and conforming to the pattern
+protocol. Pixel values are represented as 32-bit numbers encoding the
+RGBA color channels.
 
-collapse-pattern
-make-pattern-from-bitmap-file
+@code{collapse-pattern}
+@code{make-pattern-from-bitmap-file}
 
-define-bitmap-file-reader
-define-bitmap-file-writer
-bitmap-format-supported-p
-bitmap-output-supported-p
-read-bitmap-file
-write-bitmap-file
+@code{define-bitmap-file-reader}
+@code{define-bitmap-file-writer}
+@code{bitmap-format-supported-p}
+@code{bitmap-output-supported-p}
+@code{read-bitmap-file}
+@code{write-bitmap-file}
 
 Images are read with @code{read-bitmap-file}.
-Images are wrote with @code{write-bitmap-file}.
+Images are written with @code{write-bitmap-file}.
 
 Images are created with @code{make-image}.
 Images are cloned with @code{clone-image}.
 Images are copied with @code{copy-image}.
 Images are blended with @code{blend-image}.
 Images are filled with @code{fill-image}.
 
-Operations having source and destination image as arguments may use
+Operations having a source and a destination image as arguments may use
 the same image without copying it.
 
-To draw an image use @code{draw-pattern*} or @code{draw-design}. Image
-may be also used as an ink in other drawing functions.
+To draw an image use @code{draw-pattern*} or @code{draw-design}. An
+image may also be used as an ink in other drawing functions.
 
+@node Raster image as a medium
 @section Raster image as a medium
 
-Raster image backend includes a medium that implements:
+The raster image backend includes a medium that implements:
 @itemize @bullet
 @item CLIM's medium protocol,
 @item CLIM's output stream protocol, and
 @item CLIM's Pixmap protocol.
 @end itemize
 
-@anchor{Macro clime:with-output-to-image-stream}
-@deffn {Macro} {clime:with-output-to-image-stream} (stream-var file-stream format &rest options) &body body
-@findex clime:with-output-to-raster-image-stream
-@end deffn
+The extension provides a number of similar macros for controlling the
+output of images to files, streams and images:
+
+@anchor{Macro mcclim-raster-image:with-output-to-raster-image-stream}
+@anchor{Macro mcclim-raster-image:with-output-to-rgba-pattern}
+@deffn {Macro} {mcclim-raster-image:with-output-to-raster-image-stream} (stream-var stream format &rest options) &body body
+@deffnx {Macro} {mcclim-raster-image:with-output-to-rgba-pattern} (stream-var image &rest options) &body body
+@findex mcclim-raster-image:with-output-to-raster-image-stream
+@findex mcclim-raster-image:with-output-to-rgba-pattern
+Within @var{body}, @var{stream-var} is bound to a stream that produces a
+raster image.  This stream is suitable as a stream or medium argument to
+any CLIM output function, such as @code{draw-line*} or
+@code{write-string}.
```

Comment:
I would add warning here that the package will most likely change to clime in the future (or even comment whole section). These things are not well tested afaik and code needs cleanup.

Reply:
this remark applies to all macro exported from mcclim-raster-image: